### ISSUE TITLE
Incorrect Status, Name for Hurricane Alex, 2016, Adv 9

### ISSUE THREAD

**Issue by timtrice** [ropensci/rrricanes#14]
```r
fstadv("http://www.nhc.noaa.gov/archive/2016/al01/al012016.fstadv.009.shtml?")
```

**Comment by timtrice** [comment]
Also occurs for 
  * AL022016, Adv 12
  * AL022016, Adv 34
  * AL032016, Adv 9
  * AL032016, Adv 10
  * AL042016, Adv 7
  * AL052016, Adv 17
  * AL062016, Adv 27
  * AL072016, Adv 47
  * AL082016, Adv 16
  * AL092016, Adv 25:38
  * AL202016, Adv 17
  * AL112016, Adv 21
  * AL122016, Adv 45
  * AL132016, Adv 22
  * AL142016, Adv 45:47
  * AL152016, Adv 57
  * EP222016, Adv 24
  * EP022016, Adv 8
  * ...

This is most likely due to the last advisory being issued or some other comment after the advisory number in the header. Pattern needs to be modified to accept any non-\n character between the advisory number and the \n character.

**Comment by timtrice** [comment]
Corrected in commit d338503.
